src/fileseq/frameset.py

```diff
@@ -1114,6 +1124,52 @@ def _build_frange_part(start, stop, stride, zfill=0):
         else:
             return '{0}-{1}x{2}'.format(pad_start, pad_stop, stride)
 
+    @staticmethod
+    def _build_frange_part_decimal(start, stop, count, strides, min_stride, max_stride, zfill=0):
+        """
+        Private method: builds a proper and padded subframe range string from
+        decimal values.
+
+        Args:
+            start (decimal.Decimal): first frame
+            stop (decimal.Decimal): last frame
+            count (int): number of frames in range (inclusive)
+            strides (set[decimal.Decimal]): set of increments (maximum 2)
```

Comment:
I'm not clear on the purpose of a set of strides in this function. It seem to only be used once when it is checked for a length of 1, to derive a single stride value. And then otherwise it is calculated from the count and min/max stride values

Reply:
While reading `framesToFrameRange`, it seems the strides set is more of an implementation detail of the calling function, which is what confused me. It only makes sense in the context of what is calling it.

Reply:
Yeah, that's true. Maybe a `stride` argument that is None if `_build_frange_part_decimal` will need to calculate a stride value?